keyboards/40percentclub/mf68/keymaps/mf68_ble/rules.mk

```diff
@@ -5,5 +5,5 @@ F_CPU = 8000000
 #   change yes to no to disable
 #
 BLUETOOTH_ENABLE = yes
-BLUETOOTH_DRIVER = BluefruitLE
+BLUETOOTH_DRIVER = bluefruit_le
```

Comment:
aren't nested jsons in keymaps preferred? or not enforced yet?

Reply:
I'm keeping keymaps mostly untouched for now - just cleaning up core + keyboards; anything else that breaks as a result I try to unbreak.